src/help.ts

```diff
@@ -16,12 +19,33 @@ export class BBHelp {
   }
 
   public static load(config: any = {}): Promise<any> {
+    if (config.defaultHelpKey !== undefined) {
+      BBHelpClient.defaultHelpKey = config.defaultHelpKey;
+    }
+
+    config.getCurrentHelpKey = BBHelpClient.getCurrentHelpKey;
+
     return registerScript('https://cdn.blackbaudcloud.com/bb-help/bb-help.js')
       .then(() => {
-        BBHelp.addStyles();
-
+        BBHelpClient.addStyles();
         // Initialize the widget.
         BBHELP.HelpWidget.load(config);
       });
   }
+
+  public static setCurrentHelpKey(helpKey: string = BBHelpClient.defaultHelpKey): void {
```

Comment:
My brain is failing to remember... What (who?) will be calling these public methods? Your components?

Reply:
These are being called from my Service, as well as Skyux Builder.

Reply:
Builder: calling the load method.  
https://github.com/blackbaud/skyux-builder/blob/master/src/app/app.component.ts#L197 

service: https://github.com/blackbaud/skyux-lib-help/pull/5/files#diff-490473f08cd29a3d01c483951791e31cR8